docs/changes/coordinates/11601.other.rst

```diff
@@ -0,0 +1,2 @@
+The new DE440 and DE440s ephemerides are now available via shortcuts
+'de440' and 'de440s'.  The DE 440 ephemeris will become the default in 5.0.
```

Comment:
You should clarify that these are JPL ephemerides, and by "default" you mean that DE440 will become the default _JPL_ ephemeris, not the overall default ephemeris (i.e., over the built-in ephemeris).

Reply:
Good point, done!